core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -2864,6 +2867,7 @@ public String toString() {
           .setDefaultValue(false).build();
   public static final PropertyKey USER_BLOCK_WRITE_LOCATION_POLICY =
       new Builder(Name.USER_BLOCK_WRITE_LOCATION_POLICY)
+          .setAlias("alluxio.user.block.write.location.policy.class")
```

Comment:
I changed this property to be `alluxio.user.block.write.location.policy` without suffixed `class`. if you guys don't like the alias, I could remove it.

Reply:
Personally, I like the class at the end because it points to a specific implementation. It also gives more of a hint as to what the value of the property should be. The `.class` suffix is used commonly in `hdfs-site`, `mapred-site`, and `yarn-site` as well.

I don't think we should change the name of this property.

Reply:
Since the value needs to be a class name I think having the suffix is fine.

Reply:
keep `.class`, make a ticket to support passing a short policy name